README.md

```diff
@@ -322,21 +322,14 @@ You can render the HTML yourself with ReDoc:
 
 ## Updating from 1.x
 
-* `header.md` change to `body.json`
-    ```json
-      {
-        "openapi": "3.0.0",
-        "info": {
-          "title": "Dox Example",
-          "description": "description.md",
-          "contact": {},
-          "version": "2.0"
-        }
-      }
-    ```
-* update rspec config file
-  * `header_file_path` -> `body_file_path`
-* Add needed schema paths
+* remove depricated configs
+  * Dox.config.header_file_path
+* Rename new configs
+  * Dox.config.desc_folder_path -> Dox.config.descriptions_location
+* add optinal configs
+  * Dox.config.title = 'Dox Example'
+  * Dox.config.description = 'description.md'
```

Comment:
Oh, this is like a header description. Not sure if `description` is clear enough.

Reply:
this is a great idea